autoconnect/autoconnect-ws/autoconnect-ws-sm/src/unidentified.rs

```diff
@@ -92,6 +94,13 @@ impl UnidentifiedClient {
                 }
             })
             .send();
+        if let Some(chids) = channel_ids {
+            self.app_state
+                .metrics
+                .count_with_tags("busi.channel_count", chids.len() as u64)
```

Comment:
My general question about this is documenting and making clear what "busi" means, as described in the Jira ticket given it's a business, not performance concern.  However, even calling it a business concern remains a little opaque for me. Is there a plan for there to be more categorical hierarchies that fall under this sub-root in the future? I suppose we could  put out a core metric and add that sub root down the road once there are more within the category, or if more are expected this is a good first pass. 

At the least, I would suggest `business` instead of `busi`, given it's just more explicit for those who might look at it and not know what exactly it's pertaining to.  What do you think?

Reply:
Fair. It's a new category that relates more to the business logic than anything else. Using the longer form might help. Since Autopush has two main components, and therefore two different roots, there must be two different metrics.